codebuild_specs/e2e_workflow.yml

```diff
@@ -131,33 +131,34 @@ batch:
           CLI_REGION: eu-central-1
       depend-on:
         - publish_to_local_registry
-    - identifier: containers_api_secrets_schema_function_1_api_3_api_1
+    - identifier: containers_api_secrets_schema_function_1_api_3_base_cdk
```

Comment:
These changes have tried to come in w/ every PR I've opened recently, I think our workflow spec is out of date w/ reality, so I'll just let these flow in.